### DISCUSSION TITLE
Custom SQL Test

### DISCUSSION THREAD

**Discussion by ziggekatten** [open-metadata/OpenMetadata#18646] [Q&A] [answered]
So I have an case where I need to do an query on an table and if the count returns 0, it should be considered an fail. In effect, as long as count i > that 0 it should be considered passed

How do I achieve this? Using threshold only trigger failure when >than count. I would need < than count

**Comment by Vsshouver** [comment, marked as answer]
Para resolver essa lógica, você pode implementar a consulta em SQL ou na linguagem de programação que está utilizando. Aqui está um exemplo genérico em SQL:

Estrutura do exemplo:
- Suponha que você tenha uma tabela chamada tabela_exemplo.
- Você quer verificar se há registros que atendam a uma determinada condição.

`SELECT 
    CASE 
        WHEN COUNT(*) > 0 THEN 'Aprovado'
        ELSE 'Falha'
    END AS Status
FROM tabela_exemplo
WHERE <condição>;
`
Substitua <condição> pela condição que deseja avaliar.
O resultado será Aprovado se o número de registros for maior que 0 e Falha caso contrário.
